### DISCUSSION TITLE
Solar Radiation has no Icon

### DISCUSSION THREAD

**Discussion by MarkWildig** [cgiesche/streamdeck-homeassistant#245] [Help and Support] [answered]
HI Thank you for the amazing HA plugin , just found a small bug that when using solar radiation from  a Tempest weather station no icon appears 
Thank you

**Comment by cgiesche** [comment, marked as answer]
If there is no "hand made" icon defined in my plugin, it automatically tries to use the icon that comes from home-assistant:

![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/9ca7b8dc-8a38-491c-b609-300a953f4c9c)
![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/5a253aa1-7d41-483c-aabc-842f810d3ad4)

If you set an icon for your solar radiation entity in home assistant, it should appear on the stream deck button.

**Comment by MarkWildig** [comment]
Thank you so much for the information and the fantastic app

Mark

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
